data-sources/flex-install-windows-cert/config.yml

```diff
@@ -0,0 +1,26 @@
+id: flex-install-windows-cert
```

Comment:
let's rename this `windows-certs` to create consistency with the how dashboards are referred to.

Reply:
this should be done now too, would love it if you can let me know if it looks good.